test/Interpreter/existential_transform.swift

```diff
@@ -0,0 +1,31 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift -O -wmo %s -o %t/a.out
+// RUN: %target-codesign %t/a.out
+// RUN: %target-run %t/a.out | %FileCheck %s
+// REQUIRES: executable_test
```

Comment:
This REQUIRES: objc_interop too. Maybe write this test without using Foundation though? Just define your own URL struct